lib/registries/addon/drafts/draft/-components/metadata-institutions-manager/component.ts

```diff
@@ -0,0 +1,87 @@
+import { tagName } from '@ember-decorators/component';
+import Component from '@ember/component';
+import { action, computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency-decorators';
+import I18N from 'ember-i18n/services/i18n';
+import Toast from 'ember-toastr/services/toast';
+
+import { layout } from 'ember-osf-web/decorators/component';
+import Institution from 'ember-osf-web/models/institution';
+import Node from 'ember-osf-web/models/node';
+import { QueryHasManyResult } from 'ember-osf-web/models/osf-model';
+import CurrentUser from 'ember-osf-web/services/current-user';
+
+import template from './template';
+
+@tagName('')
+@layout(template)
+export default class MetadataInstitutionsManagerComponent extends Component {
+    // Required
+    node!: Node;
+
+    // private properties
+    @service i18n!: I18N;
+    @service toast!: Toast;
+    @service currentUser!: CurrentUser;
+
+    affiliatedList!: QueryHasManyResult<Institution>;
+    currentAffiliatedList!: QueryHasManyResult<Institution>;
+
+    @computed('affiliatedList.[]', 'loadNodeAffiliatedInstitutions.isRunning')
+    get fieldIsEmpty() {
+        if (this.loadNodeAffiliatedInstitutions.isRunning) {
+            return false;
+        }
+        return this.affiliatedList && !this.affiliatedList.length;
+    }
+
+    @computed('node.isRegistration')
+    get emptyFieldText() {
+        if (this.node.get('isRegistration')) {
+            return this.i18n.t('osf-components.institutions-list.no_affiliated_institution.registration');
+        }
+        return this.i18n.t('osf-components.institutions-list.no_affiliated_institution.project');
+    }
```

Comment:
Any reason you need these computed properties here? They are sort of specific to editable fields.